0MI0800256/task01.07.03.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+struct Numbers{
+    int a;
+    int b;
+    int c;
+};
+
+int read()
+{
+ifstream file("result.txt");
```

Comment:
Hard-coding the file name is a rather poor idea, don't you think ? You can pass it as an argument to the function instead :P

Reply:
btw - also a good idea - check whether the file was indeed opened :thinking: